code/modules/mob/living/carbon/human/species_types/humans.dm

```diff
@@ -29,3 +29,54 @@
 		if(prob(1))
 			return 'sound/voice/human/wilhelm_scream.ogg'
 		return pick(male_screams)
+
+/datum/species/human/prepare_human_for_preview(mob/living/carbon/human/human)
+	human.hair_style = "Business Hair"
+	human.hair_color = "b96" // brown
+	human.update_hair()
+
+/datum/species/human/get_species_description()
+	return "Humans are the dominant species in the known galaxy. \
+		Their kind extend from old Earth to the edges of known space."
+
+/datum/species/human/get_species_lore()
+	return list(
+		"These primate-descended creatures, originating from the mostly harmless Earth, \
+		have long-since outgrown their home and semi-benign designation. \
+		The space age has taken humans out of their solar system and into the galaxy-at-large.",
+
+		"In traditional human fashion, this near-record pace from terra firma to the final frontier spat \
+		in the face of other races they now shared a stage with. \
+		This included the lizards - if anyone was offended by these upstarts, it was certainly lizardkind.",
+
+		"Humanity never managed to find the kind of peace to fully unite under one banner like other species. \
+		The pencil and paper pushing of the UN bureaucrat lives on in the mosaic that is TerraGov; \
+		a composite of the nation-states that still live on in human society.",
+
+		"The human spirit of opportunity and enterprise continues on in its peak form: \
+		the hypercorporation. Acting outside of TerraGov's influence, literally and figuratively, \
+		hypercorporations buy the senate votes they need and establish territory far past the Earth Government's reach. \
+		In hypercorporation territory company policy is law, giving new meaning to \"employee termination\".",
+	)
+
+/datum/species/human/create_pref_unique_perks()
+	var/list/to_add = list()
+
+	if(CONFIG_GET(number/default_laws) == 0) // Default lawset is set to Asimov
+		to_add += list(list(
+			SPECIES_PERK_TYPE = SPECIES_POSITIVE_PERK,
+			SPECIES_PERK_ICON = "robot",
+			SPECIES_PERK_NAME = "Asimov Superiority",
+			SPECIES_PERK_DESC = "The AI and their cyborgs are, by default, subservient only \
+				to humans. As a human, silicons are required to both protect and obey you.",
+		))
+
+	if(CONFIG_GET(flag/enforce_human_authority))
+		to_add += list(list(
+			SPECIES_PERK_TYPE = SPECIES_POSITIVE_PERK,
+			SPECIES_PERK_ICON = "bullhorn",
+			SPECIES_PERK_NAME = "Chain of Command",
+			SPECIES_PERK_DESC = "Nanotrasen only recognizes humans for command roles, such as Captain.",
```

Comment:
only for HoP and Captain, other command roles have some non-human species.
Maybe include better pay as one of the perks